.github/dependabot.yml

```diff
@@ -0,0 +1,45 @@
+# See help here: https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/enabling-and-disabling-version-updates
+
+version: 2
+updates:
+  #
+  # Python dependencies
+  #
+  - package-ecosystem: pip
+    directory: "/python"
+    schedule:
+      interval: "daily"
+      time: "09:00"
+      timezone: "Europe/Dublin"
+    open-pull-requests-limit: 3
+    assignees:
+      - "apaniukov"
+    versioning-strategy: increase-if-necessary
+  #
+  # Tests
+  #
+  - package-ecosystem: pip
+    directory: "/tests"
```

Comment:
I don't see requirements files in `tests` folder

Reply:
yes, but they could be there in future...

Reply:
excluded such dirs